control-plane/roles/monitoring/templates/cluster-monitoring/alertmanager.yaml

```diff
@@ -0,0 +1,64 @@
+---
+apiVersion: v1
+kind: Service
```

Comment:
Can you explain to me why this is necessary? To me it looks like the service is deployed by the operator and configured through the kube-prometheus-stack chart. Is there something not working?